cmd/glusterfs/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	gfd "github.com/gluster/gluster-csi-drivers/pkg/glusterfs"
```

Comment:
need to change this import path `gluster-csi-drivers` to `gluster-csi-driver`

Reply:
currently building is failing for me because of this

Reply:
its based on the gopath, any way I renamed to gluster-csi-driver for parity with this repo.